### PR TITLE
Use DALI_extra images in more tests

### PR DESCRIPTION
**Pull request by JanuszL**
#### Why we need this PR?
- moves most of the Jupyter notebooks to DALI_extra to avoid redownloading heavy data from the internet or using data sources not available to the external user

#### What happened in this PR?
 - downloading Sintel trailer for every video test is a too heavy task and we can run out of download quota
 - moved jupyter examples to DALI_extra data
 - tested on CI
 - examples were update

**JIRA TASK**: DALI-1022

### CODE REVIEW

**Comment by JanuszL** [comment]
Need to fix `test_operator_crop_mirror_normalize` test and GTest that have file paths hard coded (and no longer valid).

**Comment by jantonguirao** [comment]
!build

**Comment by jantonguirao** [comment]
!build

**Comment by JanuszL** [comment]
Last commit needs to be reverted as it is for the testing purpose only.

File: qa/setup_dali_extra.sh [hunk 1]
```diff
@@ -3,16 +3,19 @@
 # Fetch test data
 export DALI_EXTRA_PATH=${DALI_EXTRA_PATH:-/opt/dali_extra}
 
-export DALI_EXTRA_URL=${DALI_EXTRA_URL:-"https://github.com/NVIDIA/DALI_extra.git"}
+#export DALI_EXTRA_URL=${DALI_EXTRA_URL:-"https://github.com/NVIDIA/DALI_extra.git"}
+
+DALI_EXTRA_URL="https://github.com/jantonguirao/DALI_extra/"
+
```
Comment (jantonguirao) [inline qa/setup_dali_extra.sh]: I believe this should be reverted

**Comment by jantonguirao** [comment]
!build

**Pull request merged**